test/IntegrationTests/StrongNamedTests.cs

```diff
@@ -27,7 +27,8 @@ public StrongNamedTests(ITestOutputHelper output)
     {
     }
 
-    [Fact]
+    // TODO: Fix the StrongNamed test
+    [Fact(Skip = "StrongNamedTests is problematic right now.")]
```

Comment:
I am fine with merging skipped tests, but it will be great to have GH issue for this, and put link to GH to Skip reason.

Reply:
Added a GH issue to address this and the other related tests: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2403